### ISSUE TITLE
url has github.io domain twice

### ISSUE THREAD

**Issue by hoolymama** [mkdocs/mkdocs#1830]
Hi there,

I have the docs folder in my repo: `org_name.github.io` in a branch called `authoring`
When I run `mkdocs gh-deploy` it prints out this: 

```
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /Users/julian/dev/conductor/org_name.github.io/site
INFO    -  Copying '/Users/julian/dev/conductor/org_name.github.io/site' to 'master' branch and pushing to GitHub.
INFO    -  Your documentation should shortly be available at: https://org_name.github.io/org_name.github.io/
```

And it is there. But I'd like it to be available at `https://org_name.github.io`

Is this possible?

Many thanks

**Comment by waylan** [comment]
Have you tried going to `https://org_name.github.io`? Does that work?

If not, what config options do you have set in your `mkdocs.yml` config file?

And what options are you passing to `mkdocs gh-deploy`?

**Comment by hoolymama** [comment]
Hi @waylan 

This problem may have magically fixed itself overnight. I first noticed it after a coworker had added a CNAME. When I went to either `docs.org_name.com` or `org_name.github.io` I'd see a 404. Only by going to the doubled-up URL mentioned earlier did I see the site.

This morning, however, everything works. 

I figured that `docs.org_name.com` didn't work because org_name.github.io` didn't work. But now I'm thinking the CNAME was actually the cause of the problem. Guessing a delay in propagation. 

Apologies for time-wasting.
